packages/cli/src/commands/proxy.ts

```diff
@@ -0,0 +1,46 @@
+import { getHttpOperationsFromResource } from '@stoplight/prism-http';
+import { pick } from 'lodash';
+import { CommandModule } from 'yargs';
+import { createMultiProcessPrism, CreateProxyServerOptions, createSingleProcessPrism } from '../util/createServer';
+import sharedOptions from './sharedOptions';
+
+const proxyCommand: CommandModule = {
+  describe: 'Start a proxy server with the given document file',
+  command: 'proxy <document> <upstream>',
```

Comment:
Having two positional args may be confusing:
1) folks won't remember which one should be first
2) their order does not result from some logical explanation: e.g. `cp <from> <to>` <- this has logical explanation

Moreover, most of the time we can implicate upstream host. It should be in OAS document.

To sum this up, I'd recommend optional: `-u / --upstream` which when not provided results in host form OAS "api description".

Reply:
+ a failure when neither -u nor oas/host is provided

Reply:
@StefanDywersant, I think that the conclusion was that `upstream` is not gonna be validated against what's in spec. cc @XVincentX

Reply:
Oh, thank you for that lovely gif :)

However, I believe our cli tools should more or less look like iphone interfaces rather than not iphone. that being said, even if you just woke up or u're drunk you should be able to spin proxy instance without looking to the manual.

Reply:
I do get the point :) I'm fine with the current version though

Reply:
I'm ok with both. @philsturgeon what do you think?